amivapi/events/model.py

```diff
@@ -181,7 +182,7 @@
             },
             'additional_fields': {
                 'nullable': True,
-                'type': 'json_schema',
+                'type': 'cerberus_schema',
```

Comment:
Ah, i see! We should definitely change this, but that has nothing to do witu this pull request then.